src/components/handlers/Selector.vue

```diff
@@ -80,6 +81,14 @@ async function load () {
 			containerAttrs: { class: ['data-container', 'column-container', 'padding'] }
 		}
 	}
+else if (tags['Content-Type'] === 'model/stl') {
+		data.loaded = true
+		return data.handler = {
+			is: markRaw(Model3D),
+			attrs: { src: gatewayLink.value + '.stl' },
```

Comment:
currently concatenating the file extension here as I wasn't being able to load the STL file in the Browser using just the typical chivesweave url with the tx.id termination